app/helpers/logo_helper.rb

```diff
@@ -0,0 +1,16 @@
+module LogoHelper
+  def show_logo_text(path)
```

Comment:
I was unsuccessful in my (albeit half-hearted) attempt to understand Ruby enough to combine this into a single function

Reply:
separating the concerns seems fine anyhow